docs/release.sh

```diff
@@ -116,17 +84,22 @@ invalidate_cache() {
 
 	aws configure set preview.cloudfront true
 
-	files=$(find site/ -not -name "*.md*" -type f | sed 's/site\///g')
+	# Get all the files
+	# not .md~ files
+	# replace spaces w %20 so urlencoded
+	local IFS=$'\n'
+	files=( $(find site/ -not -name "*.md*" -not -type f | sed 's/site//g' | sed 's/ /%20/g') )
```

Comment:
@jfrazelle you forgot to remove `-not`. And since we remove whitespaces, we shouldn't need `IFS` stuff either, but to be confirmed :)